cocos/core/root.ts

```diff
@@ -525,7 +525,6 @@ export class Root {
     public createWindow (info: IRenderWindowInfo): RenderWindow | null {
         const window = this._createWindowFun(this);
         window.initialize(this.device, info);
-        this._windows.push(window);
```

Comment:
Why remove it?

Reply:
The logic of whether to add a window depends on whether the camera is used or not. If the camera is not used, it is not necessary to add renderWindow to the windows property of the root

Reply:
Got it. Sounds reasonable.